Library/Homebrew/dev-cmd/pr-upload.rb

```diff
@@ -118,7 +118,7 @@ def pr_upload
         Upload bottles described by these JSON files to #{service}:
           #{json_files.join("\n  ")}
       EOS
-      return
+      return unless github_packages?(bottles_hash)
```

Comment:
Feels weird to print the same output and then actually do stuff. I'd suggest having an early `return` inside the `elsif` instead (so it's not checked twice, too).